salt/top.sls

```diff
@@ -133,8 +134,8 @@ base:
     - datadog.plugins
   'G@roles:elasticsearch and P@environment:(micromasters|rc-apps|production-apps)':
     - match: compound
-    - elasticsearch
-    - elasticsearch.plugins
+    - elastic_stack.elasticsearch
+    - elastic_stack.elasticsearch.plugins
```

Comment:
These two lines should remain unmodified since that is for a separate cluster which is currently deployed using the old formula.